### ISSUE TITLE
exception triggered with mlflow ui: AttributeError: 'NoneType' object has no attribute 'search_model_versions'

### ISSUE THREAD

**Issue by stablum** [mlflow/mlflow#2322]
(mlflow --version is 1.5.0)
In the web-based ui, by accessing the url of a run:
```
2020/01/19 13:06:02 ERROR mlflow.server: Exception on /ajax-api/2.0/preview/mlflow/model-versions/search [GET]
Traceback (most recent call last):
  File "/home/francesco/deepfake-detection-challenge/venv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/francesco/deepfake-detection-challenge/venv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/francesco/deepfake-detection-challenge/venv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/francesco/deepfake-detection-challenge/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/francesco/deepfake-detection-challenge/venv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/francesco/deepfake-detection-challenge/venv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/francesco/deepfake-detection-challenge/venv/lib/python3.7/site-packages/mlflow/server/handlers.py", line 137, in wrapper
    return func(*args, **kwargs)
  File "/home/francesco/deepfake-detection-challenge/venv/lib/python3.7/site-packages/mlflow/server/handlers.py", line 581, in _search_model_versions
    model_versions_detailed = _get_model_registry_store().search_model_versions(
AttributeError: 'NoneType' object has no attribute 'search_model_versions'

```

**Comment by ggliem** [comment]
Please refer to the following issue:
https://github.com/mlflow/mlflow/issues/1975

It seems like Model Registry does not support file based stores .(https://github.com/mlflow/mlflow/blob/master/mlflow/server/handlers.py#L60)

You could try using e..g postgresql to replace your file-based store. MLflow supports the database dialects mysql, mssql, sqlite, and postgresql ([Documentation](https://www.mlflow.org/docs/latest/tracking.html#logging-to-a-tracking-server)).

**Comment by mparkhe** [comment]
This issue has been addressed by in #2456 (@aarondav). Model Registry requires a SQL based backend. If not specified as `--backend-store-uri` argument, the error is handled in UI. I am closing this issue out.

![image](https://user-images.githubusercontent.com/6900999/83928637-44f96b00-a745-11ea-9487-10b3cdfce087.png)

**Comment by SergiyShebotnov** [comment]
By any chance does Model Registry support file based storage now?
